<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Update_car</class>
 <widget class="QMainWindow" name="Update_car">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>-20</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update Car</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Car ID :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_carID">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_searchIDCar">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>90</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Used Car</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Car</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>981</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_cengine">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_carname">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_color">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_factory">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>130</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_bprice">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>70</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_model">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>130</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Car Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Engine Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Existing</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>190</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Create Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Buy Price</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fatory</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_informationcar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>951</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>information Car</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_cdate">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>190</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_yes">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>yes</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_no">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_save_updatecustomer">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>460</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_violation">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>260</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Violation</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_damage">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>260</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>260</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Damage</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_km">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>200</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>KmUsed</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_save_updatecustomer_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_save_updatecustomer_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>520</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1166</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
